pkg/cloudprovider/vsphere/util.go

```diff
@@ -58,3 +60,25 @@ func ConvertK8sUUIDtoNormal(k8sUUID string) string {
 		k8sUUID[24:36])
 	return strings.ToLower(strings.TrimSpace(uuid))
 }
+
+// ErrOnLocalOnlyIPAddr returns an error if the provided IP address is
+// accessible only on the VM's guest OS.
+func ErrOnLocalOnlyIPAddr(addr string) error {
```

Comment:
Hi @dvonthenen,

Hmm, this looks familiar :)

Reply:
It is!